### PR TITLE
add page object model example

### PR DESCRIPTION
**Pull request by mdcruz**
This adds a new " Examples " section under "Using k6 browser". This will be expanded in the future to include more examples of how to use k6 browser in various use cases.

Screenshot 2023-07-20 at 16 08 06

### CODE REVIEW

File: src/data/markdown/translated-guides/en/03 Using k6 browser/100 Examples.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: 'Examples'
+excerpt: 'A list of different examples and recommended practices when working with the k6 browser module'
+---
+
+This section presents some examples and recommended practices when working with the `k6 browser` module to leverage browser automation as part of your k6 tests.
+
+- [Page object model pattern](/using-k6-browser/examples/page-object-model-pattern)
```
Comment (heitortsergent) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/100 Examples.md:8]:
```suggestion
- [Page object model pattern](/using-k6-browser/examples/page-object-model)
```

It looks like the link should be `page-object-model` here.

Comment (mdcruz) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/100 Examples.md]: Yes, thanks @heitortsergent! 💜

**Review by ka3de — APPROVED** [review]
LGTM.

**Review by ankur22 — APPROVED** [review]
LGTM 🚀

**Review by ppcano — APPROVED** [review]

**Pull request merged**